provider/resource.go

```diff
@@ -119,6 +132,21 @@ func NewResource(id, rt string, p Provider) Resource {
 	}
 }
 
+func (r *resource) AttributesReference() ([]string, error) {
+	resourceFunc := providerResources[r.provider.String()]
```

Comment:
Check the `ok` also, as it could not exists.

Reply:
@xescugc done in 6fbac852e750d05ed7db8fbf972ea43f5e522ead :)